bridges/otelzap/core_test.go

```diff
@@ -1,6 +1,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
+// Copyright (c) 2016-2017 Uber Technologies, Inc.
 package otelzap
```

Comment:
There needs to be an empty line in between. Otherwise it would be seen as package comment.

```suggestion
// Copyright (c) 2016-2017 Uber Technologies, Inc.

package otelzap
```